pkg/reconciler/revision/resources/deploy.go

```diff
@@ -228,7 +228,7 @@ func MakeDeployment(rev *v1alpha1.Revision,
 		Spec: appsv1.DeploymentSpec{
 			Replicas:                ptr.Int32(1),
 			Selector:                makeSelector(rev),
-			ProgressDeadlineSeconds: &ProgressDeadlineSeconds,
```

Comment:
why this change?

Reply:
So I can make `ProgressDeadlineSeconds` const, which allows me to make `activationTimeout` const.